Speech/src/V2/BatchRecognizeFileResult.php

```diff
@@ -32,64 +26,52 @@ class BatchRecognizeFileResult extends \Google\Protobuf\Internal\Message
      */
     private $metadata = null;
     /**
-     * The transcript for the audio file. This is populated only when
-     * [InlineOutputConfig][google.cloud.speech.v2.InlineOutputConfig] is set in
-     * the
-     * [RecognitionOutputConfig][[google.cloud.speech.v2.RecognitionOutputConfig].
+     * Deprecated. Use `cloud_storage_result.native_format_uri` instead.
+     *
+     * Generated from protobuf field <code>string uri = 1 [deprecated = true];</code>
+     * @deprecated
+     */
+    protected $uri = '';
+    /**
+     * Deprecated. Use `inline_result.transcript` instead.
      *
-     * Generated from protobuf field <code>.google.cloud.speech.v2.BatchRecognizeResults transcript = 4;</code>
+     * Generated from protobuf field <code>.google.cloud.speech.v2.BatchRecognizeResults transcript = 4 [deprecated = true];</code>
+     * @deprecated
      */
-    private $transcript = null;
```

Comment:
Changed the visibility here. Private to Protected is more relaxed access modifier. I hope it is okay